lib/pacemaker/pcmk_graph_consumer.c

```diff
@@ -766,7 +762,7 @@ pcmk__unpack_graph(const xmlNode *xml_graph, const char *reference)
 
         CRM_CHECK(buf != NULL,
                   pcmk__free_graph(new_graph); return NULL);
-        pcmk__scan_min_int(buf, &(new_graph->id), -1);
+        pcmk__scan_min_int(buf, &(new_graph->id), 1);
```

Comment:
just calling out that elsewhere we're shifting `-1` to `0` and here we're shifting it to `1` (as the default for unparsable transition ID). I think that still aligns with your intent, since the graph isn't empty.

Reply:
Exactly, 0 would be the empty graph and 1 would be the first real graph. Here we should never need it, but if there's some sort of bug or corruption in the graph, we'd use 1